jina/peapods/runtimes/remote/jinad/__init__.py

```diff
@@ -0,0 +1,40 @@
+from argparse import Namespace
+from typing import Union, Dict, Optional
+
+from jina.peapods.runtimes.remote.jinad.api import PeaAPI
+from jina.peapods.runtimes.remote import RemoteRuntime
+from jina.helper import cached_property, namespace_to_dict, colored, typename
+from .api import get_jinad_api
+
+
+class RemoteJinaDRuntime(RemoteRuntime):
+    """A RemoteJinaDRuntime that will spawn a remote `BasePea` or `BasePod` via REST communication with a jinad instance
+    """
+    def __init__(self, args: Union['Namespace', Dict], kind: str):
+        super().__init__(args, kind=kind)
+        if isinstance(self.args, Namespace):
+            self.ctrl_timeout = self.args.timeout_ctrl
+        if isinstance(self.args, Dict):
+            api_args = self.args['peas'][0]
+        else:
+            api_args = self.args
+
+        self.api = get_jinad_api(kind=self.kind, host=api_args.host, port=api_args.port_expose, logger=self.logger)
+
+    @cached_property
+    def remote_id(self) -> str:
```

Comment:
Keep consistent with `spawn_remote`
```suggestion
    def remote_id(self) -> Optional[str]:
```